src/Annotation/Entity.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @author Szyon Kunowski <szymon.kunowski@gmail.com>
+ */
+
+namespace Opstalent\CrudBundle\Annotation;
+
+use Doctrine\Common\Annotations\Annotation;
+use Opstalent\CrudBundle\Annotation\Traits\ActionTrait;
+use Opstalent\CrudBundle\Annotation\AbstractAnnotation;
```

Comment:
isn't it in the same dir? it's not needed if its in the namespace of `Opstalent\CrudBundle\Annotation`